diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8267230..66de70a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -70,11 +70,6 @@ target_link_libraries(${PROJECT_NAME}
         Libcrypt::Libcrypt
 )
 
-file(GLOB TS_FILES translations/*.ts)
-qt5_create_translation(QM_FILES ${TS_FILES})
-add_custom_target(translations DEPENDS ${QM_FILES} SOURCES ${TS_FILES})
-add_dependencies(${PROJECT_NAME} translations)
-
 install(TARGETS ${PROJECT_NAME} RUNTIME DESTINATION /usr/bin)
 
 install(FILES
@@ -82,5 +77,3 @@ install(FILES
     DESTINATION /usr/share/applications/
     COMPONENT Runtime
 )
-
-install(FILES ${QM_FILES} DESTINATION /usr/share/${PROJECT_NAME}/translations)
