diff --git a/CMakeLists.txt b/CMakeLists.txt
index a491ed9..8102b73 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -10,7 +10,7 @@ set(CMAKE_AUTORCC ON)
 set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
-set(QT Core Widgets DBus Quick QuickControls2 LinguistTools)
+set(QT Core Widgets DBus Quick QuickControls2)
 find_package(Qt5 REQUIRED ${QT})
 find_package(KF5WindowSystem REQUIRED)
 
@@ -51,10 +51,4 @@ target_link_libraries(${PROJECT_NAME}
         KF5::WindowSystem
 )
 
-file(GLOB TS_FILES translations/*.ts)
-qt5_create_translation(QM_FILES ${TS_FILES})
-add_custom_target(translations DEPENDS ${QM_FILES} SOURCES ${TS_FILES})
-add_dependencies(${PROJECT_NAME} translations)
-
 install(TARGETS ${PROJECT_NAME} RUNTIME DESTINATION /usr/bin)
-install(FILES ${QM_FILES} DESTINATION /usr/share/${PROJECT_NAME}/translations)
