diff --git a/CMakeLists.txt b/CMakeLists.txt
index a62bde7..1286636 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -11,7 +11,7 @@ set(CMAKE_AUTORCC ON)
 set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
-find_package(Qt5 CONFIG REQUIRED Widgets DBus X11Extras Concurrent Svg LinguistTools QuickControls2)
+find_package(Qt5 CONFIG REQUIRED Widgets DBus X11Extras Concurrent Svg QuickControls2)
 find_package(KF5WindowSystem REQUIRED)
 find_package(FishUI REQUIRED)
 find_package(PkgConfig REQUIRED)
@@ -116,10 +116,4 @@ target_link_libraries(cutefish-statusbar
   ${XCB_LIBS_LIBRARIES}
 )
 
-file(GLOB TS_FILES translations/*.ts)
-qt5_create_translation(QM_FILES ${TS_FILES})
-add_custom_target(translations DEPENDS ${QM_FILES} SOURCES ${TS_FILES})
-add_dependencies(cutefish-statusbar translations)
-
 install(TARGETS cutefish-statusbar RUNTIME DESTINATION /usr/bin)
-install(FILES ${QM_FILES} DESTINATION /usr/share/cutefish-statusbar/translations)
