diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6a0a7f6..12be07c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -10,7 +10,7 @@ set(CMAKE_AUTORCC ON)
 set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
-find_package(Qt5 CONFIG REQUIRED Widgets DBus X11Extras Concurrent LinguistTools QuickControls2)
+find_package(Qt5 CONFIG REQUIRED Widgets DBus X11Extras Concurrent QuickControls2)
 
 find_package(KF5WindowSystem REQUIRED)
 find_package(FishUI REQUIRED)
@@ -52,10 +52,4 @@ target_link_libraries(${PROJECT_NAME} PRIVATE
         KF5::WindowSystem
 )
 
-file(GLOB TS_FILES translations/*.ts)
-qt5_create_translation(QM_FILES ${TS_FILES})
-add_custom_target(translations DEPENDS ${QM_FILES} SOURCES ${TS_FILES})
-add_dependencies(${PROJECT_NAME} translations)
-
 install(TARGETS ${PROJECT_NAME} RUNTIME DESTINATION /usr/bin)
-install(FILES ${QM_FILES} DESTINATION /usr/share/${PROJECT_NAME}/translations)
